cmd/minikube/cmd/status.go

```diff
@@ -64,7 +65,9 @@ var statusCmd = &cobra.Command{
 				cmdUtil.MaybeReportErrorAndExit(err)
 			}
 		}
-		status := Status{ms, ls}
+		con := "Insert future config status message here."
```

Comment:
The config msg can be something like:
"kubectl: Misconfigured: pointing to stale minikube-vm at <ip>
To fix the kubectl context, run minikube update-context"
"kubectl: Configured: pointing to minikube-vm at <ip>"